File: Leet Code/Python/problem_9.py
class Solution(object):
    def isPalindrome(self, x):
        if ( x < 0):
            return False

        reverse = 0
        temp = x
        while (temp > 0):
            reverse = (reverse * 10) + (temp % 10)
            temp = temp // 10

        return (reverse == x)

myObj = Solution()
num = int(input("Enter number: "))
print(myObj.isPalindrome(num))

File: Leet Code/JavaScript/problem_9.js
const isPalindrome = function(x) {
    if ( x < 0) {
        return false;
    };
    let temp = x;
    let reverse = 0;

    while (temp > 0) {
        reverse = (reverse * 10) + (temp % 10);
        temp = Math.floor(temp / 10);
    };
    return (reverse === x);
};

result = isPalindrome(1423241);
console.log(result);